docs/source/index.rst

```diff
@@ -38,6 +38,7 @@ cartopy's documentation is arranged userguide form with reference documentation
     crs/index.rst
     crs/projections.rst
     tutorials/using_the_shapereader.rst
+    tutorials/map_styling.rst
```

Comment:
This definitely needs to happen. It is necessary to understand *how* to plot something, before you can annotate it.